src/definitions.js

```diff
@@ -625,6 +625,20 @@ const definitions = [
     type: 'String',
     credentialItem: true,
   },
+  {
+    identifier: 'cvc:SSN:number',
```

Comment:
SSN is bad pattern here, we used Camel Case in every variable. My suggestion SocialSecurityNumber or Ssn

Reply:
Solved